tests/Api.Integration.Tests/Endpoints/App/AppTests.cs

```diff
@@ -0,0 +1,200 @@
+using System.Net;
+using System.Net.Http.Json;
+using Bogus;
+using FluentAssertions;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.DependencyInjection;
+using Passwordless.Api.Integration.Tests.Helpers;
+using Passwordless.Service.Models;
+using Passwordless.Service.Storage.Ef;
+using Xunit;
+
+namespace Passwordless.Api.Integration.Tests.Endpoints.App;
+
+public class AppTests : IClassFixture<PasswordlessApiFactory>, IDisposable
+{
+    private readonly HttpClient _client;
+    private readonly PasswordlessApiFactory _factory;
```

Comment:
I tried moving the dispose method and parts out to a base class. However, this gets messy because each class configures its own client separately and factory isn't required on every test so I think I'll just keep them separate for now and maybe implement a common class later.